### ISSUE TITLE
(winforms) Drawing transparent Control (Text) while vlc is playing video

### ISSUE THREAD

**Issue by Prakash19921206** [ZeBobo5/Vlc.DotNet#205]
i'm using vlc control in a Panel(Control) then placed a label on top of vlc control. i want label's background to be transparent. i used below code

```
 label1.Parent = vlcControl1;
 label1.backColor = Color.Transparent;
```
The issue here is that label displays until vlc is not playing any media.
as soon as i play a media file, label gets hidden.
i also tried 
```
  label1.bringToFront();
  label1.Visible=true;
```
if i stop vlc from playing a video file `vlcControl1.Stop(); `label will then be visible.

how to set label on top of overlaying video?

**Comment by jeremyVignelles** [comment]
I had that kind of issues with writing a text as overlay to the video. It turned out it was much easier to do with the VLC `marq` plugin if the text doesn't change or just displays the date. It may fit your need.

**Comment by Prakash19921206** [comment]
@jeremyVignelles , Thanks for your reply. Can you point to any documentation or sample code?

**Comment by jeremyVignelles** [comment]
https://wiki.videolan.org/Documentation:Modules/marq/

Be careful, there is currently an issue with special characters which should be fixed very soon : #74 

For example, in WinForms designer, you can use (in VlcMediaPlayerOptions) : 
```
--sub-source=marq{marquee=test,color=16776960}
```

I'm closing this, please reopen if needed.
